.devops/nix/package.nix

```diff
@@ -87,6 +88,13 @@ let
     ]
   );
 
+  darwinSymlinks = runCommand "darwin-build-symlinks" {
+    __impureHostDeps = [ "/usr/bin/xcrun" ];
```

Comment:
NOTE: this will expose `/usr/bin/xcrun` to this particular `runCommand`, but not to the `mkDerivation` below (which is where you actually try to access xcrun)